### ISSUE TITLE
Line length of jinja tags is off by 2 characters I think

### ISSUE THREAD

**Issue by nfcampos** [tconbeer/sqlfmt#236]
I believe this can be fixed here? https://github.com/tconbeer/sqlfmt/blob/24e75e3984f058c9f9ab7419d72ef6a931f1c5e9/src/sqlfmt/jinjafmt.py#L342 

I think maybe the closing characters `}}` aren't being counted

See an example (with line length set to 80)

Input:
```
with
    a as (

        select
            {{
                dbt_utils.surrogate_key(
                    var("surrogate_key_columns_order_item")
                )
            }} as order_item_id,
            {{ dbt_utils.surrogate_key(var("surrogate_key_columns_menu_item")) }}
            as menu_item_id,
        from b

    )

select *
from a
```

Actual:
Measure the length of the line with `surrogate_key_columns_menu_item`. It is 82.
```
with
    a as (

        select
            {{
                dbt_utils.surrogate_key(
                    var("surrogate_key_columns_order_item")
                )
            }} as order_item_id,
            {{ dbt_utils.surrogate_key(var("surrogate_key_columns_menu_item")) }}
            as menu_item_id,
        from b

    )

select *
from a
```

Expected:
```
with
    a as (

        select
            {{
                dbt_utils.surrogate_key(
                    var("surrogate_key_columns_order_item")
                )
            }} as order_item_id,
            {{
                dbt_utils.surrogate_key(
                    var("surrogate_key_columns_menu_item")
                )
            }} as menu_item_id,
        from b

    )

select *
from a
```

**Comment by tconbeer** [comment]
Did you install the jinjafmt extra? eg `pip install shandy-sqlfmt[jinjafmt]`

This is the expected behavior without that extra

**Comment by nfcampos** [comment]
Yes I did, which is confirmed by the fact that adding a few extra characters to that line triggers formatting it

**Comment by tconbeer** [comment]
This is now fixed in main from #237 .

We were counting the `}}` -- the problem was actually that we weren't accounting for the leading whitespace properly

**Comment by nfcampos** [comment]
Ah I see. Great thank you!
